e2e/cypress/integration/interactive_dialog/scrollable_spec.js

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/**
+* Note: This test requires webhook server running. Initiate `npm run start:webhook` to start.
```

Comment:
would this be possible to force with some sort of `before` step? not sure if there is a ping-alike for this. Having a clear message can help people understand quickly why it failed

Reply:
We have this pattern on several spec files. Good point on clear message.  We can probably add check in `before` hook, like,
```javascript
cy.request(Cypress.env().webhookBaseUrl).its('status').should('equal', 200);
```